### PR TITLE
Update pass-system.md

### PR DESCRIPTION
**Pull request by thefranke**
Text speaks about three components, then lists four.

### CODE REVIEW

**Review by sptramer — APPROVED** [review]
Approved pending DCO fix.

**Comment by sptramer** [comment]
@thefranke before we can merge your pull request we require that you sign off with the [Developer Certificate of Origin](https://developercertificate.org/), our form of attribution to an individual for any given commit. To fix the DCO on this pull request, please follow the [Fix single-commit DCO instructions](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md#fix-option-1-amend-single-commit).

**Review by chanmosq — APPROVED** [review]

**Pull request merged**
